.mkosi/mkosi.arch

```diff
@@ -11,6 +11,7 @@ Distribution=arch
 [Output]
 Format=raw_btrfs
 Bootable=yes
+HostonlyInitrd=yes
```

Comment:
Not a fan of hostOnly mode (btw, why isn#t this called HostOnlyInitRd=?)

the problem is that the generated images only work on qemu then i guess...

Reply:
The option is called --hostonly-initrd on the mkosi command line. That translates to HostonlyInitrd in the config file.

I assumed these images are only meant for debugging purposes using qemu. Are you using them for anything else? If so, enabling this might not be a good idea. On the other hand, If we're only using the images with qemu, we might as well enable the option to save a second on boot time.

Reply:
hmm, you have a point, this are indeed mostly for local debugging, so this makes sense indeed.

Reply:
on the mkosi side, I guess enough warnings are warrented in the documentation for the options, especially for the empty password one

Reply:
I'll add some warnings to the docs but I would hope users using mkosi would know the associated risks of deleting the root password.